libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -433,6 +433,39 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
         }
     }
 
+    @ReactMethod
+    fun fetchReverseSwapFees(promise: Promise) {
+        try {
+            var reverseSwapFees = getBreezServices().fetchReverseSwapFees()
+            promise.resolve(readableMapOf(reverseSwapFees))
+        } catch (e: SdkException) {
+            e.printStackTrace()
+            promise.reject(TAG, e.message ?: "Error calling fetchReverseSwapFees", e)
+        }
+    }
+
+    @ReactMethod
+    fun inProgressReverseSwaps(promise: Promise) {
+        try {
+            var inProgressReverseSwaps = getBreezServices().inProgressReverseSwaps()
+            promise.resolve(readableArrayOf(inProgressReverseSwaps))
+        } catch (e: SdkException) {
+            e.printStackTrace()
+            promise.reject(TAG, e.message ?: "Error calling inProgressReverseSwaps", e)
+        }
+    }
+
+    @ReactMethod
+    fun sendOnchain(amountSat: ULong, onchainRecipientAddress: String, pairHash: String, satPerVbyte: ULong, promise: Promise) {
```

Comment:
The RN bridge doesn't like `ULong`, instead use `Double` then convert in the method e.g. `amountSat.toULong()`

Reply:
Done.